mantidproject/mantid
# Improve the user interface of WANDPowderReduction

**Description of work.**



- Related Gitlab user story: [PD129](https://code.ornl.gov/sns-hfir-scse/diffraction/powder/powder-diffraction/-/issues/129)
- This is #31154  into `master`

This PR is mostly a user interface tune-up for the current `WANDPowderReduction`:

- User can again select directly from ADS from GUI
- The candidate items for `Target` property is reduced to the three that are relevant to HB2C (Theta, ElasticQ, ElasticDSpacing)
- Change `EFixed` to wavelength with a default value of 1.4865A
- The default bin number is set to `1000` per request

**To test:**
Since this is a GUI tune-up, the best way to test is to build the branch and launch the GUI to check.
The functionality check can be done via existing unit test.










---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
